dlang.org.ddoc

```diff
@@ -242,16 +242,16 @@ $(SUBMENU
 )
 NAVIGATION_RESOURCES=
 $(SUBMENU
+    $(ROOT_DIR)acknowledgements.html, Acknowledgments,
+    https://wiki.dlang.org/Books, Books,
     $(ROOT_DIR)tools.html, D-Specific Tools,
```

Comment:
> I would just write Tools, we don't write "Books about D" or "Tutorials about D" either

The idea is that these tools were written specifically for the D language. They're not general purpose tools with D support. The latter could be expected with just "Tools" as a title, and then we disappoint people with just these four weird little programs.